packages/linter/src/migrations/update-16-8-0-add-ignored-files/update-16-8-0-add-ignored-files.spec.ts

```diff
@@ -8,6 +8,7 @@ describe('update-16-8-0-add-ignored-files migration', () => {
 
   beforeEach(() => {
     tree = createTreeWithEmptyWorkspace();
+    tree.write('.eslintrc.json', '{}');
```

Comment:
This is necessary for `isEslintConfigSupported` to work. Normally, the root eslintrc would be there